applications_tests/gls_sharp_navier_stokes_3d/rbf_sphere_mms.output

```diff
@@ -0,0 +1,67 @@
+Running on 1 MPI rank(s)...
+   Number of active cells:       64
+   Number of degrees of freedom: 500
+   Volume of triangulation:      8
+
+*****************************************************************
+Steady iteration :        1/3
+*****************************************************************
+Newton iteration: 0  - Residual:  0.67795
+  -Tolerance of iterative solver is : 0.00067795
+  -Iterative solver took : 4 steps
+		alpha =      1 res = 0.01309
+Newton iteration: 1  - Residual:  0.01309
+  -Tolerance of iterative solver is : 1.309e-05
+  -Iterative solver took : 5 steps
+		alpha =      1 res = 4.913e-06
+Newton iteration: 2  - Residual:  4.913e-06
+  -Tolerance of iterative solver is : 4.913e-09
+  -Iterative solver took : 5 steps
+		alpha =      1 res = 3.225e-09
+div u : 1.241e-08
+error at the weak BC : 0.3751
+L2 error velocity : 0.3046 L2 error pressure: 1.342
```

Comment:
We do not add verbosity in the output of test because they are generally not portable. Please remove this.

Reply:
That's right, I didn't notice. Initially I removed the linear and non-linear solver sections to keep the .prm light, but it seems that verbose is the default option.

Reply:
It takes about 94 seconds in Debug with the CI